src/test/java/com/github/javafaker/NumberTest.java

```diff
@@ -0,0 +1,92 @@
+package com.github.javafaker;
+
+import org.junit.Before;
+import org.junit.Test;
+
+import java.lang.*;
+import java.lang.Number;
+import java.math.BigDecimal;
+
+import static org.junit.Assert.*;
+
+public class NumberTest {
+
+    private Faker faker;
+
+    @Before
+    public void setUp() {
+        faker = new Faker();
+    }
+
+    @Test
+    public void testRandomDigit() {
+        for (int i = 0; i < 1000; ++i) {
+            int value = faker.number().randomDigit();
+            assertTrue(value <= 9 && value >= 0);
```

Comment:
could you use hamcrest matchers for consistency in tests for the assertions? Thanks